README.md

```diff
@@ -125,6 +125,7 @@ If a provider you're looking for does not exist, consider making a PR to add one
 | BattleNet | [![NuGet](https://buildstats.info/nuget/AspNet.Security.OAuth.BattleNet?includePreReleases=false)](https://www.nuget.org/packages/AspNet.Security.OAuth.BattleNet/ "Download AspNet.Security.OAuth.BattleNet from NuGet.org") | [![MyGet](https://buildstats.info/myget/aspnet-contrib/AspNet.Security.OAuth.BattleNet?includePreReleases=true)](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.BattleNet "Download AspNet.Security.OAuth.BattleNet from MyGet.org") | [Documentation](https://develop.battle.net/documentation/guides/using-oauth "BattleNet developer documentation") |
 | Beam (Mixer) | [![NuGet](https://buildstats.info/nuget/AspNet.Security.OAuth.Beam?includePreReleases=false)](https://www.nuget.org/packages/AspNet.Security.OAuth.Beam/ "Download AspNet.Security.OAuth.Beam from NuGet.org") | [![MyGet](https://buildstats.info/myget/aspnet-contrib/AspNet.Security.OAuth.Beam?includePreReleases=true)](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.Beam "Download AspNet.Security.OAuth.Beam from MyGet.org") | [Documentation](https://dev.mixer.com/reference/oauth "Mixer developer documentation") |
 | Bitbucket | [![NuGet](https://buildstats.info/nuget/AspNet.Security.OAuth.Bitbucket?includePreReleases=false)](https://www.nuget.org/packages/AspNet.Security.OAuth.Bitbucket/ "Download AspNet.Security.OAuth.Bitbucket from NuGet.org") | [![MyGet](https://buildstats.info/myget/aspnet-contrib/AspNet.Security.OAuth.Bitbucket?includePreReleases=true)](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.Bitbucket "Download AspNet.Security.OAuth.Bitbucket from MyGet.org") | [Documentation](https://developer.atlassian.com/bitbucket/api/2/reference/meta/authentication "Bitbucket developer documentation") |
+| Bitrix24 | [![NuGet](https://buildstats.info/nuget/AspNet.Security.OAuth.Bitrix24?includePreReleases=false)](https://www.nuget.org/packages/AspNet.Security.OAuth.Bitrix24/ "Download AspNet.Security.OAuth.Bitrix24 from NuGet.org") | [![MyGet](https://buildstats.info/myget/aspnet-contrib/AspNet.Security.OAuth.Bitrix24?includePreReleases=true)](https://www.myget.org/feed/aspnet-contrib/package/nuget/AspNet.Security.OAuth.Bitrix24 "Download AspNet.Security.OAuth.Bitrix24 from MyGet.org") | [Documentation](https://training.bitrix24.com/rest_help/oauth/index.php "Bitrix24 developer documentation") &#124; [Claims](/src/AspNet.Security.OAuth.Bitrix24/Claims.md) |
```

Comment:
`Claims.md` should just be content in a `bitrix24.md` file in the `docs` folder that follows the format used for the existing documentation: https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/tree/dev/docs